package/src/editor/api/index.js

```diff
@@ -301,5 +301,27 @@ export const EditorApi = Object.extend(
 
     this.commonPageConfigurationTabs.apply(view);
     return view;
+  },
+
+  /**
+   * Ensures that Editor only runs on modern browsers.
+   *
+   * @callback start
+   *  callback function to start the app.
+   * @function browserNotSupportedView
+   *  View to display in case of a not supporting browser.
+   * @function isBrowserSupported
+   *  Checks to see if the browser is supported.
+  */
+  ensureBrowserSupport: function(start) {
```

Comment:
Since this method is not part of the public API (it is only ever called by `startEditor`), it does not need inline doc. The public API we extended actually is [`registerEntryType`](https://github.com/codevise/pageflow/blob/49e310350cabd8e78e5f349034521b44fb79812a/package/src/editor/api/index.js#L96). Let's document these options there.

Reply:
I should have written: Since this method is not part of the public API, it is not supposed to show up in the [API docs](https://codevise.github.io/pageflow-docs/js/master/index.html) and thus must not have an inline comment (since everything with an inline comment shows up in the docs.